src/kochka.com.mx/src/Homepage.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from 'react';
+import sx from '@adeira/sx';
+import fbt from 'fbt';
+
+import HomepageFooter from './HomepageFooter';
+import Logo from './Logo';
+import HomepageNavigation from './HomepageNavigation';
+import SocialMediaIcons from './SocialMediaIcons';
+
+export default function Homepage(): React.Node {
+  return (
+    <div className={styles('background')}>
+      <div className={styles('backgroundOverlay')}>
+        <HomepageNavigation />
+        <main id="main" className={styles('root')}>
+          <div className={styles('center')}>
+            <Logo />
+
+            <div>
+              {/* TODO: popups explaining what is each of the elements (photo) */}
+              <fbt desc="catch phrase">REAL CATS + DUMPLINGS + COFFEE</fbt>
```

Comment:
One could say all items referring to a menu :-) How about something like:
"COFFEE + DUMPLINGS + CUTE CATS AROUND"

Reply:
Thanks for the input! Open-source FTW! 🙂 We are going to discuss it and decide later.